Automattic/wp-calypso
# Customer Home: Remove editor deprecation task

#### Changes proposed in this Pull Request

Now that the Calypso editor has been deprecated, we can remove the task
card from the Customer Home. The server code has already been removed,
so the card will not be returned as part of the view.

#### Testing instructions

Slightly tricky to test as it's about removing code, but with sites where you have the Classic editor preference set (both Simple and Atomic, but not Jetpack), visit the Customer Home screen (`/home/`) and check that the main task at the top is not about the editor deprecation, and that there are no errors/warnings related to the Customer Home view, in the browser's developer console.
